custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py

```diff
@@ -408,7 +420,8 @@ def create_partition(self, state_id):
 
     def aggregation_queries(self):
         return [
+            """INSERT INTO "{new_tablename}" (SELECT * FROM "{tmp_tablename}")""".format(new_tablename=self.new_tablename, tmp_tablename=self.temporary_tablename),
             'DROP TABLE IF EXISTS "{monthly_tablename}"'.format(monthly_tablename=self.monthly_tablename),
-            'ALTER TABLE "{tmp_tablename}" RENAME TO "{monthly_tablename}"'.format(monthly_tablename=self.monthly_tablename, tmp_tablename=self.temporary_tablename),
+            """ALTER TABLE "{new_tablename}" RENAME TO \"{tablename}\"""".format(new_tablename=self.new_tablename, tablename=self.monthly_tablename),
```

Comment:
nit: I think you does need `\` before `"`

Reply:
I don't because of the triple quotes. The one at the end only did so that the first 3 of the 4 in a row were not interpreted as the end.